staging/src/k8s.io/apiserver/pkg/apis/audit/types.go

```diff
@@ -153,6 +153,10 @@ type Policy struct {
 	// The default audit level is None, but can be overridden by a catch-all rule at the end of the list.
 	// PolicyRules are strictly ordered.
 	Rules []PolicyRule
+
+	// OmitStages is a list of stages for which no events are created. Note that this can also
+	// be specified per rule in which case the union of stages is skipped.
```

Comment:
> in which case the union of stages is skipped.

What does this mean?

Reply:
Changed to `the union of OmitStages in policy and per rule` ?

Reply:
I think what you mean (correct me if I'm wrong) is:
```
// OmitStages is a list of stages for which no events are created. Note that this can also
// be specified per rule in which case the union of both are omitted.
```

I think I was confused by the use of skipped vs. omitted.